crawl-ref/source/monster.cc

```diff
@@ -1357,6 +1357,9 @@ static bool _is_signature_weapon(const monster* mons, const item_def &weapon)
         if (mons->type == MONS_MENNAS)
             return get_weapon_brand(weapon) == SPWPN_HOLY_WRATH;
 
+        if (mons->type == MONS_VAUD)
+            return get_weapon_brand(weapon) == SPWPN_HOLY_WRATH;
```

Comment:
This means that he won't stick with his starting weapon if it rolls the 10% chance of flaming, right? Seems like you should either include that here too or should remove the flaming chance from the weapon table.